<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenEditView</class>
 <widget class="QWidget" name="RenEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { border: 1px solid blue; background: qlineargradient(spread:pad, x1:0.466, y1:0, x2:0.465955, y2:0.688, stop:0.125 rgba(171, 174, 221, 232), stop:0.670455 rgba(255, 255, 255, 255)); }</string>
  </property>
  <widget class="QLineEdit" name="lineEditFIO">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>621</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEditS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEditE">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRent">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Арендовать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxPark">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNumber">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
